pkg/flare/archive_dca.go

```diff
@@ -61,11 +63,11 @@ func createDCAArchive(fb flaretypes.FlareBuilder, confSearchPaths map[string]str
 
 	getLogFiles(fb, logFilePath)
 	getConfigFiles(fb, confSearchPaths)
-	getClusterAgentConfigCheck(fb)             //nolint:errcheck
-	getExpVar(fb)                              //nolint:errcheck
-	getMetadataMap(fb)                         //nolint:errcheck
-	getClusterAgentClusterChecks(fb)           //nolint:errcheck
-	getClusterAgentDiagnose(fb, senderManager) //nolint:errcheck
+	getClusterAgentConfigCheck(fb)                        //nolint:errcheck
```

Comment:
Do we know the circumstances when any of these function calls might return an error? Would it be worth it to log any errors here, or are they known / expected to fail and we want to avoid generating any noise?

Reply:
@zhuminyi : Any thoughts?